src/iop/temperature.c

```diff
@@ -1366,9 +1366,13 @@ static void gui_sliders_update(struct dt_iop_module_t *self)
   if(FILTERS_ARE_CYGM(img->buf_dsc.filters))
   {
     dt_bauhaus_widget_set_label(g->scale_r, NULL, _("green"));
+    gtk_widget_set_tooltip_text(g->scale_r, _("green channel value on a scale from 0 to 8"));
```

Comment:
In my view, the range 0 to 8 needs not to be given in the tool tip. As far as I can tell, no tool tip in darktable does.

Reply:
Sure. Copied directly from manual, no problem in dropping it.